### ISSUE TITLE
high accuracy model with weak heatmap

### ISSUE THREAD

**Issue by hamedbehzadi** [jacobgil/keras-grad-cam#26]
Hi
I have trained a model which has several convolution and batch normalization layers and 2 classifier layers. This model trained with 5000 medical images categorized to 2 classes. Know, I have high accuracy and ROC but my heatmap doesn't show the right area of each class in images. I used grad-cam. Anybody knows what is the problem?

Thanks in advanced
